lib/services/local/users.go

```diff
@@ -1215,6 +1215,62 @@ func (s *IdentityService) GetSAMLAuthRequest(id string) (*services.SAMLAuthReque
 	return &req, nil
 }
 
+// CreateSSODiagnosticInfo creates new SAML diagnostic info record.
+func (s *IdentityService) CreateSSODiagnosticInfo(ctx context.Context, authKind string, authRequestID string, entry types.SSODiagnosticInfo) error {
+	if authRequestID == "" {
+		return trace.BadParameter("missing parameter authRequestID")
+	}
+
+	switch authKind {
+	case types.KindSAML, types.KindGithub, types.KindOIDC:
+		// nothing to do
```

Comment:
```suggestion
```

Reply:
The comment is there to indicate this is an explicit choice to do nothing. A seasoned reader will recognize a pattern, but a comment will make it unambiguous for everyone, including less experienced readers.